packages/compiler-cli/ngcc/src/dependencies/esm_dependency_host.ts

```diff
@@ -93,7 +225,7 @@ export class EsmDependencyHost extends DependencyHostBase {
  * in this file, true otherwise.
  */
 export function hasImportOrReexportStatements(source: string): boolean {
-  return /(import|export)\s.+from/.test(source);
+  return /(?:import|export)[\s\S]+?(["'])(?:(?:\\\1|.)*?)\1/.test(source);
```

Comment:
Nice :+1:

There is one unneeded set of parenthesis (afaict), but not really important:
```suggestion
  return /(?:import|export)[\s\S]+?(["'])(?:\\\1|.)+?\1/.test(source);
```

Reply:
I'm a bit frightened to touch that again :-)